src/advect1d_vertical.jl

```diff
@@ -0,0 +1,253 @@
+#### This Julia file contains functions to implement 
+####  1) 1-D L94 vertical advection: advect1d_l94() (Lin et al., 1994)
+####  2) 1-D PPM vertical advection: advect1d_ppm() (Collela and Woodward, 1984)
+####  The vertical advection conserve pi*q rather than q. (Lin and Rood, 1996)
+
+#### I will upload the environment later but basically the code block below is the packages I use.
+##   -------------------------------------------------------------------------------------------
+##   ]activate EarthSciML_Advection
+##   using EarthSciData, EarthSciMLBase, GasChem,
+##   DomainSets, ModelingToolkit, DifferentialEquations, Dates
+##   using Plots, Statistics, GeoDataFrames, NCDatasets
+##   using Flux
+##   using BSON: @save
+##   using BSON: @load
+##   using Base.Threads
+##   using LoopVectorization
+##   using Tullio
+##   -------------------------------------------------------------------------------------------
+
+
+
+
+## L94 advection in 1-D
+## This is to make splitted scheme
+function advect1d_vertical_L94(s1d_in,vel_1d,nz,Δt,p)
```

Comment:
Add a documentation string for this function, for example like this one: https://github.com/EarthSciML/EarthSciMLBase.jl/blob/1392c624f6d63ccbdb5d0852b8b5130baf57d4f8/src/operator_compose.jl#L18

(and also for other functions)

Reply:
Also include a reference to the paper describing this algorithm.

Reply:
Also make sure to define what the function arguments are (e.g. what does `s1d_in` mean) in the documentation string.